mycli/main.py

```diff
@@ -693,6 +694,9 @@ def get_prompt(self, string):
         string = string.replace('\\d', sqlexecute.dbname or '(none)')
         string = string.replace('\\t', sqlexecute.server_type()[0] or 'mycli')
         string = string.replace('\\n', "\n")
+        string = string.replace('\\D', datetime.now().strftime('%a %b %d %H:%M:%S %Y'))
+        string = string.replace('\\p', socket.gethostbyname(socket.gethostname()))
```

Comment:
@amjith You're correct. I misread the MySQL documentation. Stopped at "The current TCP/IP" 😆 . Thanks! Will fix this.